tests/functional/u/use/used_before_assignment_py37.py

```diff
@@ -14,7 +14,7 @@ def second_correct_typing_method(self, other: List[MyClass]) -> bool:
         return self == other[0]
 
     def incorrect_default_method(
-        self, other=MyClass() # [used-before-assignment]
+        self, other=MyClass() # [undefined-variable]
```

Comment:
:+1: It matches the real error netter as it's `NameError: name 'MyClass' is not defined`